lib/features/academic_calendar/utils/localize_counter_message.dart

```diff
@@ -0,0 +1,17 @@
+import "package:flutter/cupertino.dart";
+
+import "../../../utils/context_extensions.dart";
+import "../model/academic_calendar_extensions.dart";
+import "../repository/academic_calendar_repo.dart";
+
+extension LocalizeCounterMessageExtension on AcademicCalendar {
```

Comment:
missing X in the name I guess?

Reply:
Sure, I also refactored other parts of code here that didn't use that naming convention